### DISCUSSION TITLE
Cron automation email not working

### DISCUSSION THREAD

**Discussion by Cnote43** [Budibase/budibase#9156] [Help]
I'm self-hosting using the single docker container.

I've been battling with this for about a month but haven't gotten it to work. I'm trying to get my app to send an email every day at 4pm. It will successfully pass the test and then (after re-publishing) it will send the email the next time 4pm rolls around, but it won't send any future emails.

I'm using **0 16 * * \*** as the cron entry. I tried switching to one of the presets (**0 8 * * \***) but still have the same problem. The "Automation History" log has always remained empty. I don't have any other automations configured besides this cron email automation.

What am I missing?

**Comment by dewdeded** [comment]
You have to (re)publish the app to make (configuration) changes to automations effective.

**Reply by Cnote43** [reply]
Yep. That's what I meant when I wrote "after re-publishing." I do that every time, but the problem persists.

**Comment by dewdeded** [comment]
Do you see your automations in the Bull Dashboard?

http://<ip>:4001/bulladmin
(in the bbapps-container in the multi docker image deployment)

Does any cron preset work? Does the automation history gets repeated entries, if u run the cron every 5 minutes?

The cron trigger works for me in the multi docker container deployment and in the single container deployment.

**Comment by Cnote43** [comment]
I only have the single container deployment. I created a new automation to send an email every minute, using the same settings as my daily cron automation, except no HTML in the message body. That worked. I wish the logs were giving me some clue about what's happening with my daily automation but there's nothing. It will successfully send an email when I test it and it will successfully send an email the first time the cron gets triggered after publishing, but then it won't send any more.

**Comment by Cnote43** [comment]
Thought I would try reviving this thread to see if anyone has any other thoughts. The problem is the same. My cron expression is **0 16 * * \***. It will send the email when I test the automation, and it will automatically send an email the next time the cron expression is true, but it will never send it again. I republish the app every time after testing it, so that's not the problem. Using a cron expression for every minute will work, but not one for once a day.

**Reply by melohagan** [reply]
Hey @Cnote43 

What does your automation look like? Is it simply that cron trigger and a send email action. Anything else happening in the automation? Do you have any other automations running?

**Reply by Cnote43** [reply]
Thanks for trying to help, @melohagan. Although I don't have a solution yet I think I've determined that the problem is due to me running this on Unraid, which doesn't have a traditional cron setup. Consider my issue resolved from the budibase perspective.
